siliconcompiler/toolscripts/ubuntu24/install-keplerformal.sh

```
@@ -0,0 +1,72 @@
+    pkg-config bison flex doxygen libspdlog-dev libfmt-dev libboost-iostreams-dev zlib1g-dev
+
+sudo apt-get install -y git
+
+mkdir -p deps
+cd deps
+
+python3 -m venv .keplerformal --clear
+. .keplerformal/bin/activate
+python3 -m pip install cmake==3.31.6
+
+git clone $(python3 ${src_path}/_tools.py --tool keplerformal --field git-url) keplerformal
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)